products/impulse/README.md

```diff
@@ -0,0 +1,91 @@
+# helpwave impulse
+
+This document includes all the necessary information to understand the idea and concepts behind helpwave impulse. 
+
+## The idea
+
+The aim of helpwave impulse is to reduce the load on the health care sector by helping people to live a healthier lifestyle. This will reduce the occurrence of illness, which leads to a more sustainable population. Healthier in this context means an improvement of the nutrition, fitness or mental health.
+
+## The concept
+
+### Understanding the individual needs
+
+A short Questionnaire should used to evaluate the needs of a healthier lifestyle for a user. This is import because each user has an individual lifestyle which creates the problem to choose the right task to great a healthier baseline. The Questions should evaluate the needs in the dimension’s nutrition, fitness and mental health. A user should not be encouraged to appropriate bad behaviour related to their lifestyle with this app.
```

Comment:
A short Questionnaire is used to evaluate the needs of a healthier lifestyle for a user. This is important because each user has an individual lifestyle which creates the problem to choose the right task to increase personal health. The Questions should evaluate the needs in the dimensions nutrition, fitness and mental health. A user should not be encouraged to pick up unhealthy behaviour related to their lifestyle with this app.